.gitignore

```diff
@@ -6,4 +6,5 @@
 /doc/
 /pkg/
 /spec/reports/
+*.sw?
```

Comment:
Sorry. Autocorrect sucks.

"Press not" was meant to read please note. "Fine"should have been fixed.